### ISSUE TITLE
I do not know what happened, why is it wrong???

### ISSUE THREAD

**Issue by 289202471** [osate/osate2#1925]
## Summary

The Osate always said that my behavior annex was wrong, but I think it is correct.
## Expected and Current Behavior

![Error](https://user-images.githubusercontent.com/29940173/62026755-7abcf180-b20e-11e9-961d-0f5f480eb64c.png)
The error message:
The required feature 'initialState' of 'org.osate.ba.aadlba.impl.BehaviorAnnexImpl@4c6518be{platform:/resource/Demo/lsrl1.aadl#/0/@ownedPublicSection/@ownedClassifier.1/@ownedAnnexSubclause.0/@parsedAnnexSubclause}' must be set
## Steps to Reproduce

1.new an AADL project
2.new an AADL package
3.use the model I given

```aadl
package lsrl1
public
	with Base_Types;
	thread test
		features
			p_in:in event data port Base_Types::integer;
			p_out:out event data port Base_Types::integer;
	end test;
	
	thread implementation test.default
		subcomponents
			x:data Base_Types::integer;
			annex behavior_specification{**
				states
				s0:initial complete state;
				transitions
				T_1:s0-[p_in?(x)]->s0{p_out!(x+1);};
				**};
	end test.default;
end lsrl1;
```

## Environment

* **OSATE Version**:Osate 2.5.0
* **Operating System**: Win10

**Comment by lwrage** [comment]
@Etienne13 Could you check if this is a bug, please?

**Comment by Etienne13** [comment]
This issue  is the same as #1863 and should be fixed after pull request #1875 is integrated in osate. 

Note: after the issue is fixed, you should see an error message saying the condirion in a transition out of a complete state must be a dispatch condition (on dispatch...)
